### DISCUSSION TITLE
Only root sees mount

### DISCUSSION THREAD

**Discussion by wuast94** [trapexit/mergerfs#1188] [Q&A] [answered]
i mounted via fstab "/mnt/vault:/mnt/newvault /mnt/mergedvault fuse.mergerfs cache.files=auto-full,dropcacheonclose=true,category.create=mfs,moveonenospc=true,minfreespace=3G,fsname=mergedvault,nonempty 0 0"

i can see the mount with root but dont with other users, i cant even see the folder

running on ubuntu server 
mergerfs version: 2.33.5

**Comment by trapexit** [comment, marked as answer]
You are using an old version of the software. As such you should use the documentation from that version. You have to use `allow_other` argument.

If you use the latest release it sets it automatically.

**Reply by wuast94** [reply]
❯ lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 23.04
Release:        23.04
Codename:       lunar

i try the jammy now becouse its the latest one provided. seems to work

but anyway, how to stay up to date? i have to monitor the releases page or rss feed right?

**Reply by trapexit** [reply]
As I mention in the docs Debian is *not* a rolling release distro so it will almost always be out of date. They don't keep things up to date on purpose.

You can use github watch for releases. I don't want to have a PPA because I don't want mergerfs being updated automatically. Working and old is better than updated and breaking.
